sys/test_utils/netdev_ieee802154_minimal/shell_commands.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (C) 2022 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @ingroup     test_utils_netdev_ieee802154_minimal
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for netdev Eth minimal test utility module
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "iolist.h"
+#include "kernel_defines.h"
+#include "net/netdev/ieee802154.h"
+#include "net/ieee802154.h"
+#include "net/l2util.h"
+#include "od.h"
+#include "shell.h"
+
+#include "netdev_ieee802154_minimal_internal.h"
+#include "test_utils/netdev_ieee802154_minimal.h"
+#include "init_dev.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+static char _addr_str[IEEE802154_LONG_ADDRESS_LEN * 3];
+
+static int send(int iface, le_uint16_t dst_pan, uint8_t *dst_addr,
+                size_t dst_len, char *data);
+
+int ifconfig_list(int idx)
+{
+    int res;
+    uint8_t tmp[IEEE802154_LONG_ADDRESS_LEN];
+    netdev_t *netdev = _devices[idx].dev;
+    netdev_ieee802154_t *dev = container_of(netdev, netdev_ieee802154_t, netdev);
+
+    netopt_enable_t enable_val;
+    uint16_t u16_val;
+
+    printf("Iface %3d  HWaddr: ", idx);
+    res = netdev_ieee802154_minimal_get(netdev, NETOPT_ADDRESS, tmp, IEEE802154_SHORT_ADDRESS_LEN);
+    l2util_addr_to_str(tmp, IEEE802154_SHORT_ADDRESS_LEN, _addr_str);
+    printf("%s", _addr_str);
+
+    printf(", Long HWaddr: ");
+    res = netdev_ieee802154_minimal_get(netdev, NETOPT_ADDRESS_LONG, tmp, IEEE802154_LONG_ADDRESS_LEN);
```

Comment:
```suggestion
    res = netdev_ieee802154_minimal_get(netdev, NETOPT_ADDRESS_LONG, tmp,
                                        IEEE802154_LONG_ADDRESS_LEN);
```

Again a naughty > 100 char line :)